backend/src/managers/PostManager.ts

```diff
@@ -244,12 +245,20 @@ export default class PostManager {
         const commentRaw = await this.commentRepository.createComment(userId, postId, parentCommentId, content, language, parseResult.text);
         this.userManager.clearUserRestrictionsCache(userId);
 
-        for (const mention of parseResult.mentions) {
+        let parentAuthor: UserInfo | undefined;
+        if (parentCommentId) {
```

Comment:
When there is no `parentComment` - this is a top-level comment. Should we use post's author as parentAuthor to avoid duplicate mentions?

Reply:
Post author currently doesn't receive mentions for the comments.